macro/Cargo.toml

```diff
@@ -23,5 +23,7 @@ proc-macro = true
 codec = { package = "parity-scale-codec", workspace = true }
 darling = { workspace = true }
 proc-macro-error = { workspace = true }
-syn = { workspace = true }
+syn.workspace = true
```

Comment:
I do wonder, did this happen as a result of some linting extensions from your IDE?

We could revert this to keep consistency with the dependencies of this file (`syn = { workspace = true }`)